pipelines/concourse.yml

```diff
@@ -305,6 +306,24 @@ jobs:
   on_success: *fixed-concourse
   on_failure: *broke-concourse
 
+- name: containerd-integration
+  public: true
+  max_in_flight: 1
```

Comment:
just realized we had similar concurrency limits on `testflight` and `watsjs` - obviously they run in containers so they should be independent/reproducible - are they super expensive to run? maybe because they're privileged? i really don't have a solid grasp on this.